server/aap_public/__init__.py

```diff
@@ -0,0 +1,5 @@
+from flask import Blueprint
```

Comment:
We're no longer using flask but rather quart. This import works because of the [quart-flask-patch](https://github.com/superdesk/superdesk-core/blob/async/setup.py#L70) that we're using, but the idea is to eventually remove it. So it would be better to import from quart directly now

Reply:
Cool will change in the next PR!